pom.xml

```diff
@@ -70,7 +71,7 @@
     <license.years>${project.inceptionYear}-2022</license.years>
     <license.mailto>mailto:info AT sonarsource DOT com</license.mailto>
 
-    <artifactsToPublish>${project.groupId}:${project.artifactId}:hpi</artifactsToPublish>
+    <artifactsToPublish>${project.groupId}:${project.artifactId}:hpi${project.groupId}:${project.artifactId}:json:cyclonedx</artifactsToPublish>
```

Comment:
Sorry, I have no idea, but is this correct? Shouldn't there be a `,` or a `:` after `:hpi`?

Reply:
good point, thanks

Reply:
Looking at our Ant Scanner, I think so. Missing an `,`.